wikimedia/wikipedia-ios
# Watchlist - Allow thanking user from Watchlist view

**Phabricator:** https://phabricator.wikimedia.org/T335606 (partially)

### Notes
Enables thanking a user from the Watchlist view. Note this lightly depends on https://github.com/wikimedia/wikipedia-ios-components/pull/35 which removes the thank button for changes by anonymous and bot users.

### Test Steps
1. Delete your app (or reset UserDefaults) to reset onboarding
2. Open Watchlist, and tap a username and thank them
3. Confirm the Thank onboarding modal is displayed and that when Send Thanks is tapped, a success toast shows at bottom of screen
4. Confirm subsequent thank attempts don't show onboarding
